lib/replicate/checkpointer.js

```diff
@@ -5,24 +5,60 @@ var explain404 = require('./../deps/explain404');
 var pouchCollate = require('pouchdb-collate');
 var collate = pouchCollate.collate;
 
-function updateCheckpoint(db, id, checkpoint, returnValue) {
+var REPLICATION_ID_VERSION = 2;
```

Comment:
Does PouchDB has any analogue of CouchDB instance UUID? If it has, then better raise version to 3 which is current one.